testsuite/src/main/java/io/netty5/testsuite/transport/socket/SocketAutoReadTest.java

```diff
@@ -15,11 +15,9 @@
  */
 package io.netty5.testsuite.transport.socket;
 
+import io.netty.buffer.Unpooled;
```

Comment:
Kind of unexpected that this is still used.

Reply:
Looks unused in this file. Will remove.